program1.txt

```diff
@@ -1 +1,8 @@
-hello everyone
\ No newline at end of file
+class Sample
```

Comment:
dudee put access specifiers

Reply:
I have updated the code as per ur request